doc/Development_Documentation/19_Development_Tools_and_Details/07_Logging.md

```diff
@@ -52,14 +52,30 @@ You can add your own logging functionality using Pimcore's log writer. You can c
 function like this:
 
 ##### Custom log entry
-```php
-\Pimcore\Log\Simple::log($name, $message);
+To create a custom log entry, we need to create the monolog log channels and log handlers configuration. Here is an example on how to log in a custom filename called `custom.log`
+
+```yaml
+monolog:
+    handlers:
+        custom_handler:
+            level:    debug
+            type:     stream
+            path:     '%kernel.logs_dir%/custom.log'
+            channels: [custom_log]
+    channels: [custom_log, some_other_channel]
+
 ```
+Now, when injecting the `Psr\Log\LoggerInterface` instead of using the name `$logger` like in previous examples, something like `$customLogLogger` needs to be used. Symfony will automatically figure out which channel to log into by the name of the variable.
```

Comment:
```suggestion
It is possible to inject the `Psr\Log\LoggerInterface` by changing the variable name eg. `$customLogLogger` (camel case channel name + `Logger`) and Symfony will automatically wire the specified channel.
```

suggesting some small changes, there's no other "previous examples" on the same page, it could be that it "landed" on this page first etc...